tests/storage/test_appservice.py

```diff
@@ -297,7 +299,9 @@ def test_complete_appservice_txn_updates_last_txn_state(
         service = Mock(id=self.as_list[0]["id"])
         events = [Mock(event_id="e1"), Mock(event_id="e2")]
         txn_id = 5
-        self._set_state(self.as_list[0]["id"], ApplicationServiceState.UP)
+        self.get_success(
```

Comment:
Also not sure if this needs to be here? :shrug:

Reply:
Look reasonable!